### DISCUSSION TITLE
Does 0.6.8 EFI resolve the serial number issue?

### DISCUSSION THREAD

**Discussion by MarkMorrow** [duszmox/Thinkpad-T460s-macOS-OpenCore#3] [Q&A] [answered]
I'd resisted upgrade after rolling back from prior version because of that.

**Comment by duszmox** [comment, marked as answer]
I didn't check in the previous version whether I had this problem or not, but in OC version 0.6.8 my SN seems to be the same as the one I entered in the config.plist

![Screen Shot 2021-04-08 at 18 33 04](https://user-images.githubusercontent.com/38438001/114063578-108c9900-9899-11eb-80ff-2879fcc01417.png)
(It's a temp SN on the picture, not my main one)

**Reply by MarkMorrow** [reply]
Thanks for confirming.  I upgraded and concur that your 0.6.8 EFI does display serial number the same as was the case with v0.6.5 (and unlike 0.6.7).  Nice to be running on latest version.  I'm sure there are advantages I have yet to appreciate, but very much appreciate your work and the sudo approval from apple watch piece you provided.

**Reply by MarkMorrow** [reply]
@duszmox BTW:  Tried to send you some coffee funds as a thank you with the "buy me a coffee" link off your fork.  Unfortunately it returns, "Something's not right. This page doesn't exist."

**Reply by duszmox** [reply]
Thanks so much, I think the previous link that I created somehow expired, but now it should work as expected
